usecases/auth/authentication/apikey/client.go

```diff
@@ -107,7 +107,16 @@ func (c *Client) isTokenAllowed(token string) (int, bool) {
 	return -1, false
 }
 
-func (c *Client) getUser(pos int) string {
+func (c *StaticApiKey) getUser(pos int) string {
+	// passed validation guarantees that one of those options will work
+	if pos >= len(c.config.Users) {
+		return c.config.Users[0]
+	}
+
+	return c.config.Users[pos]
+}
+
+func (c *StaticApiKey) Create(pos int) string {
```

Comment:
These functions now do the same thing, right? If so, should the `Create` impl change to simply call `getUser`?

Reply:
Create wasn't used, probably leftover when playing around. Will delete!